frontend/src/components/pages/ReviewPage.tsx

```diff
@@ -126,35 +169,67 @@ const ReviewPage = () => {
               setTranslatedStoryLines={setTranslatedStoryLines}
               numApprovedLines={numApprovedLines}
               setNumApprovedLines={setNumApprovedLines}
+              isReviewable={stage === "REVIEW"}
               reviewPage
             />
           </Flex>
-          <Flex margin="20px 30px" justify="flex-start" alignItems="center">
-            <Box marginRight="10px">
-              <ProgressBar
-                percentageComplete={
-                  (numTranslatedLines / translatedStoryLines.length) * 100
-                }
-                type="Translation"
-              />
-            </Box>
-            <Box>
-              <ProgressBar
-                percentageComplete={
-                  (numApprovedLines / translatedStoryLines.length) * 100
-                }
-                type="Review"
-              />
-            </Box>
+          <Flex margin="20px 30px" justify="space-between" alignItems="center">
+            <Flex justify="flex-start" alignItems="right">
+              <Box marginRight="10px">
+                <ProgressBar
+                  percentageComplete={
+                    (numTranslatedLines / translatedStoryLines.length) * 100
+                  }
+                  type="Translation"
+                />
+              </Box>
+              {/* BUG: Need to make these the same width */}
```

Comment:
hmmm what happens if you remove the outer `Flex` on line 177? Lets make sure we dont introduce any bugs into the code space

Reply:
![image](https://user-images.githubusercontent.com/54864799/136667122-40b23303-f820-4266-9f96-889f98d76de8.png)
LOL